frontends/precinct-scanner/src/api/scan.test.ts

```diff
@@ -11,23 +11,31 @@ test('calibrate success', async () => {
   expect(fetchMock.done()).toBe(true);
 });
 
-test('getExport returns CVRs on success', async () => {
+test('getExportWithoutImages returns CVRs on success', async () => {
   const fileContent = electionWithMsEitherNeitherFixtures.cvrData;
   fetchMock.postOnce('/precinct-scanner/export', fileContent);
-  const cvrsFileString = await scan.getExport();
+  const cvrsFileString = await scan.getExportWithoutImages();
   const lines = cvrsFileString.split('\n');
   const cvrs = lines.flatMap((line) =>
     line.length > 0 ? (JSON.parse(line) as CastVoteRecord) : []
   );
   expect(cvrs).toHaveLength(100);
 });
 
+test('getExportWithoutImages specifies to skip images in request', async () => {
+  fetchMock.postOnce(
+    { url: '/precinct-scanner/export', body: { skipImages: true } },
+    'anything'
+  );
+  await scan.getExportWithoutImages();
```

Comment:
This doesn't appear to actually validate anything about the request.

Reply:
Wouldn't the test fail if `fetchMock` was not matched?

Reply:
`fetchMock` expecting a request meeting a certain filter was, in my mind, what was being validated.

Reply:
Ah, you're right. The `body` of the first argument does match the request body.

Reply:
Do you think expecting `fetchMock.done()` would make it any more clear?

Reply:
Nah, we do this "just call a thing and if it doesn't blow up it passes" elsewhere.